debian/rules

```diff
@@ -18,7 +18,7 @@ override_dh_install:
 
 # Build our documentation as well
 override_dh_auto_configure:
-	dh_auto_configure -- --enable-js-doc
+	dh_auto_configure -- --enable-js-doc --enable-gtk-doc
```

Comment:
It's a bit sad that we have to bring gtk-doc in here only to take it out later, but this does indeed seem like the best thing to do.